src/main/java/DeadlineTask.java

```diff
@@ -0,0 +1,16 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class DeadlineTask extends Task {
```

Comment:
It's better to have (even if it's just a one-liner) description of the class.

Reply:
Thank you for the feedback! I've edited and added the respective JavaDocs as per your comments to follow the style convention more accurately. :)